packages/react-components/react-nav-preview/library/src/components/NavDrawer/useNavDrawer.ts

```diff
@@ -16,7 +16,9 @@ import type { NavDrawerProps, NavDrawerState } from './NavDrawer.types';
  * @param ref - reference to root HTMLDivElement of NavDrawer
  */
 export const useNavDrawer_unstable = (props: NavDrawerProps, ref: React.Ref<HTMLDivElement>): NavDrawerState => {
-  const focusAttributes = useArrowNavigationGroup({
+  const { tabsterDomAttribute: tabsterDomAttributeFromProps } = props;
+
+  const defaultFocusAttributes = useArrowNavigationGroup({
     axis: 'vertical',
     circular: true,
```

Comment:
Should we spread tabsterDomAttributes in here instead so that it's additive instead of replacement?

Reply:
yeah - we could do that too. Wilco :-)